src/pages/hegel/guides/general-division-of-the-logic.mdx

```diff
@@ -0,0 +1,101 @@
+
+## Further Remarks
+
+### On the Term Element
+
+Hegel highlights the term _element_ when discussing the unity of the doctrines. As George di Giovanni notes in his translation, element has classical connotations harkening back to the four elements of "air", "water", "earth" and "fire" where it is understood that something has an underlying nature of its own _but also_ has implied  modes of being or relationships to other elements (Hegel, 2010, 39). In Aristotle, we find element defined as: 
+
+> An element, we take it, is a body into which other bodies may be analysed, present in them potentially or in actuality (which of these, is still disputable), and not itself divisible into bodies different in form (Aristotle, 2001, On the Heavens, Book III, Chapter III). 
+
+As far as Aristotle was concerned then, an element cannot be divided into bodies of a different form, such that if one were to divide "air", one would not receive "fire" or "earth". Whether this is still valid for Hegel is not clear here, but what is clear is that logic principally understood as element persists throughout its many divisions or transformations and is therefore _the_ unity. 
+
+However, what might contrast to the thinking of element as a kind of "building block" is the fact that the logical developments of the concept are non-regressive. That is to say, the progression of thought cannot be reverted back to its constituents. As Hegel confirms:
+
+> For since the division is ... the _parting_ or the _judgment_ of the concept &ndash; is the positing of the determination which is already immanent in it and therefore the positing of its distinction &ndash; this positing must not be understood as resolving that concrete unity back into its determinations, as if these were to exist on their own, for this would be here a vacuous return to the previous standpoint, to the opposition of consciousness. But this opposition has vanished; the unity remains the element, and the distinctions of the division and of the development in general no longer transgress that unity (Hegel, 2010, 39/21.45). 
+
+Positing&mdash;here meaning putting something as explicit&mdash;develops the content further and reveals new details about it. This further account cannot be said to be part of the constituents as they were but must be part of them as they develop. In the context of thought and consciousness, the latter is oriented around an subject-object opposition whereas in thought, by contrast, this opposition has _vanished_, leaving logic as the sole element of which any further distinctions and classifications form moments. 
```

Comment:
I think this is very interesting and worth including. But I think that framing it beneath this discussion of the term "element" shifts the focus onto a niche, technical, discussion, and away from a helpful point, from the point of view of someone coming to this for the first time, about how they should thing about the moments of the Logic.

Reply:
It's put under "Further Remarks", don't what else do to. It's too big for a footnote. Alternative is we make an file on the Reference section for more niche technical discussions from the introductions? However, that needs to be prepared properly and so it won't be something done right away - I say we include this as is and then plan on making a garbage collector for technical niches from the Intros?